lib/src/hydrated.dart

```diff
@@ -137,40 +115,37 @@ class HydratedSubject<T> extends Subject<T> implements ValueStream<T> {
 
   @override
   void onAdd(T event) {
-    _wrapper = SubjectValueWrapper(value: event);
+    _subject.value = event;
```

Comment:
Would it be a bit more readable?
```suggestion
    _subject.add(event);
```